dev-infrastructure/Makefile

```diff
@@ -3,11 +3,11 @@ SHELL = /bin/bash
 CURRENTUSER=$(shell az ad signed-in-user show | jq -r '.id')
 LOCATION?=westus3
 RESOURCEGROUP?=aro-hcp-${AKSCONFIG}-$(USER)
+REGIONAL_RESOURCEGROUP?=aro-hcp-$(LOCATION)-$(USER)
 MAESTRO_INFRA_RESOURCEGROUP?=aro-hcp-svc-cluster-$(USER)
```

Comment:
we should rename all the resource groups to have the region names or short region names in them so if we wanted to create duplicate dev infra in multiple regions within a single sub it would work.  (plus they're easier to search with region in the name)